src/Window.cpp

```diff
@@ -714,9 +756,11 @@ void Window::onBuildExit(int exitCode, QProcess::ExitStatus exitStatus) {
     m_buildProcess = nullptr;
 }
 
+
 void Window::startRun() {
 
     // Only one algo running at a time
+    m_speedSlider->setValue(SPEED_SLIDER_DEFAULT);
```

Comment:
Convince me that this is always the correct behavior. (I'm tempted to believe that we should just leave the slider where the user put it.)

Perhaps if we want to automatically manipulate the speed at which the robot moves, we should have a separate mechanism for that, rather than hijacking this one? That way we don't lose the user's intent when they set the slider.